packages/j/jsonpath-dchester.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "jsonpath",
+  "filename": "jsonpath.min.js",
+  "description": "Query JavaScript objects with JSONPath expressions. Robust / safe JSONPath engine for Node.js.",
+  "homepage": "https://github.com/dchester/jsonpath#readme",
+  "keywords": [
+    "jsonpath",
+    "dchester",
+    "JSONPath",
+    "json-path",
+    "object",
+    "traversal",
+    "json",
+    "path",
+    "data structures",
+    "popular"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dchester/jsonpath.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "npm",
+    "target": "jsonpath",
+    "fileMap": [
+      {
+        "basePath": ".",
```

```suggestion
        "basePath": "",
```
If we need the root, the `basePath` can be empty

---

Changed